_pages/mobile-interview-project.md

```diff
@@ -89,63 +142,109 @@ Feel free to either:
-When you come in for your virtual onsite interview, you will work with two engineers over the course of two pair programming sessions (1 hour each) to add more functionality to your application. In the first pair programming session, we’ll focus on extending the data model. In the second pair programming, we’ll focus on extending the UI.
 
-During the interview, the interviewer will also ask you about general architecture choices you made while writing the app. Please be prepared to answer these questions as they come up!
+When you come in for your virtual onsite interview, you will work with two engineers over the course of two pair programming sessions (1 hour each) to add more functionality to your application. **Because of this, please don’t build more features than we outline here!**
+
+During the interview, the interviewer will also ask you about architecture choices you made while building the app. Please be prepared to answer these questions as they come up!
 
 # FAQ
-**Q) How long should I spent on the solution / how long should the solution take?**
-We estimate the solution should take 3-4 hours. Feel free to split up this time as best it works for you.
 
-**Q) Do I need to worry about separate mobile and tablet experiences?**
-No – feel free to focus on either mobile or tablet – just note which one you’re focusing on.
+ * Q) **What are you looking for? What is the right answer?**
+
+There isn’t one right answer! We’re really looking to see how you approach the problem.
+
+Very broadly, we’re looking at the quality, structure, maintainability, testability of your code, how well you’ve captured all the requirements, how well the app runs and behaves, and what your strengths in mobile app development are.
 
-**Q) What quality should the code in the final app be?**
-Treat the code as if you’re merging it to master in a real app. Take as few shortcuts / hacks as possible. If you do need to take shortcuts, please note as such in a code comment. This also applies to architecture and testability: we’re looking for code as close to the real world as possible: utilize architecture and testability best practices. As mentioned: there is no minimum or maximum amount of test coverage we’re looking for – instead try to provide enough coverage such that regressions are caught by automated tests. Only worry about unit tests – you can skip view snapshot or integration tests.
+We really want to see what “production-ready” mobile code looks like to you. If you had to compromise for the sake of speed, let us know in your README like you would in a PR review!
+
+ * Q) **How long should I spend on the solution / how long should the solution take?**
+
+We estimate the solution should take 4-6 hours. Many candidates say they took around 5 hours. Feel free to split up this time as best works for you. If you really enjoy the project, feel free to spend some more time, but we do ask that you stop after 8 hours to avoid spending too much time.
```

Comment:
I think this doesn't align w/ the time at the very top

Reply:
Thhanks for catching 💯